opennms-doc/guide-install/src/asciidoc/text/basic-setup-opennms/setting-up-opennms.adoc

```diff
@@ -0,0 +1,54 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[gi-set-up-opennms]]
+== Setting up a basic {opennms-product-name}
+
+The _{opennms-product-name}_ platform can be installed to multiple OS families.
```

Comment:
The _{opennms-product-name}_ platform can be installed **on** multiple OS families.